docs/dev/developer_guide/overview.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+## Key Technologies
+
+Galaxy NG relies on the following dependencies:
+
+- [Pulp](https://pulpproject.org/): software repository (more on this bellow)
```

Comment:
below (not 'bellow') https://proofed.com/writing-tips/word-choice-below-vs-bellow/#:~:text=Below%20is%20a%20preposition%20or,level%2C%20layer%2C%20or%20rank.&text=Bellow%20is%20a%20verb%20that,tool%20used%20to%20strengthen%20fire.